tasks/sep/020-holocene-system-config-upgrade-multi-chain/README.md

```diff
@@ -0,0 +1,42 @@
+# Holocene Hardfork Upgrade - `SystemConfig`
+
+Status: DRAFT, NOT READY TO SIGN
+
+## Objective
+
+Upgrades the `SystemConfig` for the Holocene hardfork for Sepolia/{OP,Mode,Metal,Zora,Base}
+
+The proposal was:
+
+- [ ] Posted on the governance forum.
+- [ ] Approved by Token House voting.
+- [ ] Not vetoed by the Citizens' house.
+- [ ] Executed on OP Mainnet.
+
+The governance proposal should be treated as the source of truth and used to verify the correctness of the onchain operations.
+
+Governance post of the upgrade can be found at <placeholder>.
+
+This upgrades the `SystemConfig` in the
+[op-contracts/v1.8.0](https://github.com/ethereum-optimism/optimism/tree/op-contracts/v1.8.0-rc.1) release.
+
+## Pre-deployments
+
+- `SystemConfig` - `0x29d06Ed7105c7552EFD9f29f3e0d250e5df412CD`
```

Comment:
How can I verify this address is correct?

Reply:
Yes I just spotted this too, I think we need to investigate the tooling used to deploy that contract to understand the discrepancy.

Reply:
I used the address from the logs here https://github.com/ethereum-optimism/optimism/issues/13009#issuecomment-2493469236. Looks like @sebastianst specified this address when doing his ops, such that the `SystemConfig` was never deployed.

I can redeploy the correct version of this contract and then update this PR.

Reply:
Updated here [f7080e0](https://github.com/ethereum-optimism/superchain-ops/pull/378/commits/f7080e0113dda3a00e984f01a822733db0fe6d9f). 

You can validate this on etherscan https://sepolia.etherscan.io/address/0x33b83E4C305c908B2Fc181dDa36e230213058d7d#code

Reply:
Thanks nice I have double check this is the correct implementation now.